cachito/workers/pkg_managers/general_js.py

```diff
@@ -34,6 +36,32 @@
 log = logging.getLogger(__name__)
 
 
+def get_git_prefixes():
```

Comment:
I considered doing that but we need 2 different set of git_prefixes and http_prefixes in npm.py.
I thought it might be better to define them at one place and have if else logic in the function since in the future if we're adding more to the list, we have to do it at a single place only. Wdyt?